janusgraph-core/src/main/java/org/janusgraph/diskstorage/keycolumnvalue/MultiSlicesQuery.java

```diff
@@ -0,0 +1,51 @@
+// Copyright 2017 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.diskstorage.keycolumnvalue;
+
+import org.janusgraph.diskstorage.StaticBuffer;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Queries for a list of slices of data, each identified by a start points (inclusive) and end points (exclusive).
+ * Returns all {@link StaticBuffer}s that lie in this ranges up to the given limit.
+ * <p>
+ * If a MultiSlicesQuery is marked <i>static</i> it is expected that the result set does not change.
+ *
+ * @author Sergii Karpenko (sergiy.karpenko@gmail.com)
+ */
+
+public class MultiSlicesQuery {
+
+    private final List<SliceQuery> queries;
```

Comment:
Hi!
I am having trouble implementing `org.janusgraph.diskstorage.keycolumnvalue.KeyColumnValueStore#getKeys(org.janusgraph.diskstorage.keycolumnvalue.MultiSlicesQuery, org.janusgraph.diskstorage.keycolumnvalue.StoreTransaction)` 
`MultiSlicesQuery` encapsulates completely its `queries` without any way to access them.
How is this supposed to be used ?

Reply:
Definitely need to add getter ))

Reply:
Thanks, @kptfh !

Reply:
What storage do you want to use?

Reply:
I wanted to update Janus from 0.5.3 to 0.6.0 in Apache Atlas and this class showed the problem: https://github.com/apache/atlas/blob/926fb297c13574e761db4798763780715af2477e/graphdb/janus-hbase2/src/main/java/org/janusgraph/diskstorage/hbase2/HBaseKeyColumnValueStore.java

Reply:
Hmm...
It shouldn't affect you. 
What the problem do you have?

Reply:
The only possible way to implement the new method is:
```
 @Override
    public KeySlicesIterator getKeys(MultiSlicesQuery multiSlicesQuery, StoreTransaction storeTransaction) throws BackendException {
        throw new UnsupportedOperationException("No way to use multiSlicesQuery");
    }
```